website/snippets/_legacy-sl-callout.md

```diff
@@ -1,11 +1,10 @@
-:::important Upgrade to access the new dbt Semantic Layer
+:::important Upgrade to access the latest dbt Semantic Layer
```

Comment:
`Latest` indicates that we will be upgrading/versioning the semantic layer regularly. Is that what we'll be doing moving forward?

Reply:
good question and i think we'd probably be doing that for metricflow (which powers the SL). i was told previously to use 'latest' or 'more recent' instead of 'new' when talking about a newly launched features because the word 'new' will get outdated soon.